modules/Assay/annotationProperty.csv

```diff
@@ -14,7 +14,7 @@ isStranded,Whether or not the library is stranded. (Note: TRUE = Yes; FALSE = No
 readPairOrientation,The relative orientation of the reads in a paired-end protocol,"inward, outward, matching, fr-firststrand",,FALSE,,ngs,,,,readPairOrientation,Property,Assay,annotationProperty,ngsParameter,Read_Pair_Orientation,,
 readStrandOrigin,The strand from which the read originates in a strand-specific protocol,"forward, reverse",,FALSE,,ngs,,,,readStrandOrigin,Property,Assay,annotationProperty,ngsParameter,Read_Strand_Origin,,
 libraryPrep,The general strategy by which the library was prepared,"rRNAdepletion, polyAselection, lncRNAenrichment, miRNAenrichment","libraryPreparationMethod,nucleicAcidSource,dissociationMethod",FALSE,,ngs,,,,libraryPrep,Property,Assay,annotationProperty,ngsParameter,Library_Prep,,
-libraryPreparationMethod,Method by which library was prepared,"10x, CEL-seq, Drop-Seq, Smart-seq2, TruSeq, Smart-seq4, Omni-ATAC, NEBNext mRNA Library Prep Reagent Set for Illumina, IDT xGen Exome Research Panel, TruSeq standard total RNA library kit, Illumina TruSeq DNA Nano, KAPA HyperPrep Kit PCR-free",,FALSE,,ngs,,,,libraryPreparationMethod,Property,Assay,annotationProperty,ngsParameter,Library_Preparation_Method,,
+libraryPreparationMethod,Method by which library was prepared,"10x, CEL-seq, Drop-Seq, Smart-seq2, TruSeq, Smart-seq4, Omni-ATAC, NEBNext mRNA Library Prep Reagent Set for Illumina, IDT xGen Exome Research Panel, TruSeq standard total RNA library kit, Illumina TruSeq DNA Nano, KAPA HyperPrep Kit PCR-free,KAPA RNA HyperPrep Kit with RiboErase (HMR)",,FALSE,,ngs,,,,libraryPreparationMethod,Property,Assay,annotationProperty,ngsParameter,Library_Preparation_Method,,
```

Comment:
I think `GTAC@WUSTL in-house prep` needs to be added in the valid values here. It's not in the `rangeIncludes` of the json-ld, of course. Reminds me to try to implement https://github.com/nf-osi/nf-metadata-dictionary/issues/135 so annoyances like this will go away. https://github.com/nf-osi/nf-metadata-dictionary/blob/4a6cf2dc74b07f60746a7f67e802503c8c685234/NF.jsonld#L5935-L5975

Reply:
😵 good catch!! I will add this now. Thanks!